### ISSUE TITLE
Tutorial notebook with Pamap2 dataset

### ISSUE THREAD

**Issue by dafnevk** [NLeSC/mcfly#61]
Make clear tutorial notebook that applies mcfly on the pamap2 dataset.

**Comment by vincentvanhees** [comment]
Done, except:

**Code quality open for improvement**
I compiled the tutorial notebook based on snippets from Dafne's previous notebooks. I then put all the pre-processing code in a Python function and wrote tests for it. Although the build passes, Travis indicates that there is space for improvement of code quality: adhering to naming conventions and code complexity. I am not sure how critical this is. Christiaan - I suggest that we review this together when you are back from holiday.

**Extra save and load functionality needed?**
While finalizing the notebook tutorial I realized that we do not have a clear way of storing and re-loading our trained models. Do we want to teach the end-user how to do this with the build-in Keras function like model.to_json() in combination with model.save_weights()? my suggestion would be to try to wrap this in another mcfly function, but maybe good to first discuss Christiaan after the weekend.
